src/mixer/audio.ts

```diff
@@ -327,6 +354,9 @@ export class AudioEngine extends ((EngineEmitter as unknown) as {
   newsEndCountdownEl: HTMLAudioElement;
   newsEndCountdownNode: MediaElementAudioSourceNode;
 
+  // Headphones
+  headphonesNode: GainNode;
```

Comment:
Possibly a more generic name? e.g. for multi-person shows (lol) I'd be sending guests the "headphones feed"

Reply:
Hmm, I was expecting anything going to the headphones to go through this node, so I thought this _was_ generic...

Reply:
Fair.